nibabel/cmdline/tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+""" Test scripts
+
+Test running scripts
+"""
+
+from numpy.testing import (assert_almost_equal,
+                           assert_array_equal)
+
+from nose.tools import (assert_true, assert_false, assert_raises,
+                        assert_equal, assert_not_equal)
+
+from nibabel.cmdline.utils import *
+
+
+class TestObject:
+    def __init__(self, test=None):
+        self.test = test
+
+    def get_test(self):
+        return self.test
+
+test = TestObject()
+test.test = 2
```

Comment:
please move it into `test_safe_get` so its purpose is clear and it doesn't get picked up by nose etc

Reply:
Yes, good idea.

Reply:
done and ready to rumble